### DISCUSSION TITLE
看一看我国刑事案件数据 | /project/court-data/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#646] [Announcements]
https://yuanfan.rbind.io/project/court-data/

**Comment by hongtaoh** [comment]
blogdown 很好很强大，居然可以直接支持交互式结果。我前几天做的一个[交互式图标](https://hongtaoh.com/cn/2024/11/14/power-alpha-errors/#%E4%BA%A4%E4%BA%92%E5%9B%BE%E8%A1%A8) 是我吭哧吭哧从 observable 导过去的。不知道 quarto 是不是可以直接弄。

**Reply by earfanfan** [reply]
>(97.06005402318992, 102.93994597681008)

你的博客里，像上面这样给出计算结果的框右上角也有 copy code。

>不知道 quarto 是不是可以直接弄。

我瞎猜一下，可以。

**Reply by earfanfan** [reply]
话说一个学计算机的男银为撒开始学统计学基础知识来着？我只记得你之前在博客里提到去实习做了些和 AI 大模型有关。

**Reply by hongtaoh** [reply]
>你的博客里，像上面这样给出计算结果的框右上角也有 copy code。

是，因为我是用 jupyter notebook 写，然后用[自己写的脚本](https://github.com/hongtaoh/hupyter)转成 hugo post，所以稍显粗糙。但够用。

>为撒开始学统计学基础知识

因为这学期在上一门基础统计课。我很早就想把这些东西彻底搞懂，而不是只是会用几个 R 函数。所以就逼自己一把，开始写相关的帖子。

**Reply by earfanfan** [reply]
>我很早就想把这些东西彻底搞懂，而不是只是会用几个 R 函数。

你想要的“彻底搞懂”，你自己知道是要到哪一步吗？或者说，学到哪里，你会觉得足够了。即便只是概率论与数理统计，也不浅。对于非统计学专业的学生来说，统计学是一门课程，不过对统计学专业的学生来说，统计学是一门学科了。

**Reply by hongtaoh** [reply]
很好的问题。我大意了，我说的「彻底搞懂」并非像统计学家那样精通。我说的搞懂是，不仅仅知道怎么算，还知道为什么这么算。举个具体的例子。传统的统计教学中，至少对于非统计专业的人，讲到 T 分布，就直接扔给你一个概率密度函数，然后讲一下怎么用。我觉得这非常无厘头。再比如，独立样本的 T 分布，大多数也是直接扔给你结果，证明过程全部省略。再比如，根据统计功效和显著度求样本量，也是从来不给证明过程，直接扔给你结果。阴谋论一点，我觉得这种教学模式的目的就是让人学不会，体验不到学习的乐趣，所以最后真正懂的人只是少数，这样这些老师才能继续混口饭吃。

**Reply by earfanfan** [reply]
也可能任课老师自己也不知道，于是念 PPT。我清楚地记得，我们学统计学基础课程的时候，像数学分析、高等代数、立体解析几何、常微分方程这几门课，那些老师自己很懂，于是讲的很好，尤其后两者我特别喜欢坐第一排听，等到了应用时间序列分析这门课的时候，任课老师给其他老师吐槽自己也不懂，上课用的课件是临时找到的，讲的内容就全是念念念，剩下的靠俺们自学了。

这个教学模式好像没撒特别的目的，因为促成这种教学模式的制度因素在别处。

**Reply by hongtaoh** [reply]
是的。我觉得是这样。老师如果懂的话就可以教得好。但大多数是不怎么懂的老师。
